test/test_object.py

```diff
@@ -0,0 +1,63 @@
+"""
+NOTE: this tests are also meant to be run as PyPy "applevel" tests.
+
+This means that global imports will NOT be visible inside the test
+functions. In particular, you have to "import pytest" inside the test in order
+to be able to use e.g. pytest.raises (which on PyPy will be implemented by a
+"fake pytest module")
+"""
+from .support import HPyTest
+
+
+class TestObject(HPyTest):
+    def test_getitem(self):
+        import pytest
+        mod = self.make_module("""
+            HPy_DEF_METH_O(f)
+            static HPy f_impl(HPyContext ctx, HPy self, HPy arg)
+            {
+                HPy key, result;
+                key = HPyLong_FromLong(ctx, 0);
```

Comment:
I think it would be better to use a non-common value here, instead of `0`, to minimize the chances that a test passes "by mistake" in case somewhere someone uses a default value or something like that

Reply:
Being addressed in #27.